p4-16/psa/psa.p4

```diff
@@ -40,10 +40,9 @@ typedef bit<16> EgressInstance_t;
 typedef bit<48> Timestamp_t;
 typedef error   ParserError_t;
 
+const   PortId_t         PORT_RECIRCULATE = 254;
 const   PortId_t         PORT_CPU = 255;
 
-// typedef bit<unspecified> InstanceType_t;
-// const   InstanceType_t   INSTANCE_NORMAL = unspecified;
```

Comment:
These lines look obsolete to me, as both things are defined differently later in psa.p4 now.